example/cumulus-tf/main.tf

```diff
@@ -163,21 +163,17 @@ module "cumulus" {
   # Archive API settings
   token_secret = var.token_secret
   archive_api_users = [
-    "aortega527",
+    "chuang14",
+    "ds_jennifertran",
     "jasmine",
     "jennyhliu",
     "jmccoy_uat",
-    "kbaynes",
     "kkelly",
     "kovarik",
-    "lfrederick",
-    "matthewsavoie",
-    "mboyd",
     "menno.vandiermen",
     "mobrien84",
+    "nnageswa",
     "npauzenga",
-    "ds_jennifertran",
-    "vnguyen"
```

Comment:
Should `vnguyen` be included? Seems like this might have been added in the rds-phase-3 branch?